enterprise/frontend/src/embedding-sdk/cli/steps/generate-component-files.ts

```diff
@@ -20,18 +28,24 @@ export const generateReactComponentFiles: CliStepMethod = async state => {
     ];
   }
 
-  let path: string;
+  const isNextJs = await checkIsInNextJsProject();
+
+  let reactComponentPath: string;
 
   // eslint-disable-next-line no-constant-condition -- ask until user provides a valid path
   while (true) {
-    path = await input({
+    const defaultComponentPath = isNextJs
+      ? GENERATED_COMPONENTS_DEFAULT_PATH_NEXTJS
+      : GENERATED_COMPONENTS_DEFAULT_PATH;
```

Comment:
Next.js does not have a `src/` directory, so we should import from `components/analytics` instead of `src/components/analytics` for example.